rasa/engine/recipes/default_recipe.py

```diff
@@ -220,6 +220,7 @@ def graph_config_for_recipe(
             train_schema=GraphSchema(train_nodes),
             predict_schema=GraphSchema(predict_nodes),
             training_type=training_type,
+            assistant_id=config.get("assistant_id"),
```

Comment:
This is a small thing, but we should start to gradually add small unit tests which would protect us from accidentally changing something like this.

Reply:
Could you please clarify what needs to be protected from accidental changes? The constant name?

Reply:
Added a unit test for this method, let me know if this is what you had in mind: https://github.com/RasaHQ/rasa/pull/12019/commits/6ece56025282dbcb0c62b3136be33cef96566f16

Reply:
We should test if the assignment was done properly. We can add tests for this gradually.